radix-engine/src/transaction/transaction_executor.rs

```diff
@@ -264,25 +263,6 @@ impl ExecutionConfig {
     }
 }
 
-pub struct SubstateBootStore<'a, S: SubstateDatabase> {
```

Comment:
This was unused, so I deleted it as part of the fixing of compiler warnings from 1.80